src/bap_mc/bap_mc.ml

```diff
@@ -170,6 +199,20 @@ module Cmdline = struct
     Arg.(value & opt_all ~vopt:`bil (enum formats) [] &
          info ["show-bil"] ~doc)
 
+  let addr =
+    let doc = "Specify an address of first byte, as though \
+	       the instructions occur at a certain address, \
+	       and accordingly interpreted. " in
+    Arg.(value & opt ~vopt:"0" (string) "0" &
```

Comment:
When I used baptop to test out Addr.of_string on some samples, it interprets a lack of 0x prefix as being decimal, and with it as being hex. So in each case it will function, but for clarity, I've also edited the doc for this option so that users won't have to look up that spec. :)

Dang, I've missed the spec in each case. I never saw that 0x prefix was a requirement, or I would have added it. The doc for [Addr.of_string](http://binaryanalysisplatform.github.io/bap/api/v0.9.6/experimental/codoc/index.html#/module:Std/type:addr) doesn't mention those...

Anyways, advice taken, appreciated and update applied.